apoorvakumar

```diff
@@ -0,0 +1,10 @@
+sum=0
```

Comment:
Though this is a simple program, writing comments makes you coding practice better. 
Like In the beginnning, you can have a block, where you say who the author is and the date the file is written.
Example : 
'''
Created on October 22, 2018
@author: anoopbhatn
'''

The author is your github username